src/pages/_app.js

```diff
@@ -212,7 +214,25 @@ TreetrackerApp.getInitialProps = async (context) => {
 
   const device = userAgentFromString(userAgent)?.device.type || 'desktop';
 
-  return { props, device };
+  let config = defaultConfig;
+  if (!process.env.NEXT_PUBLIC_SERVER_CONFIG_DISABLED) {
```

Comment:
@RubenSmn are you sure this works for `=false` ? I mean, does next convert the string `false` to bool?

Reply:
You're right thanks for pointing this out!